tests/app/main/test_frameworks.py

```diff
@@ -4960,47 +4962,43 @@ def test_api_is_called_to_agree(self):
+        regular_email = mock.call('email@email.com', 123456789, personalisation={'framework_name': 'g-cloud-8'})
+        mocked_notify_client.send_email.assert_has_calls([boss_email, regular_email], any_order=True)
 
-        mandrill_send_email.assert_called_once_with(
-            mock.ANY,  # self
-            to_email_addresses=['bigboss@email.com', 'email@email.com'],
-            email_body=mock.ANY,
-            subject='G-Cloud 8: you have accepted the proposed contract variation',
-            from_email_address='enquiries@digitalmarketplace.service.gov.uk',
-            from_name='Digital Marketplace Admin',
-            tags=['g-cloud-8-variation-accepted'],
-        )
-
-    @mock.patch('app.main.views.frameworks.DMMandrillClient.send_email', autospec=True)
-    def test_only_one_email_sent_if_user_is_framework_contact(self, mandrill_send_email):
+    @mock.patch('app.main.views.frameworks.DMNotifyClient', autospec=True)
+    def test_only_one_email_sent_if_user_is_framework_contact(self, mocked_notify_class):
         same_email_as_current_user = self.good_supplier_framework.copy()
         same_email_as_current_user['frameworkInterest']['declaration']['primaryContactEmail'] = 'email@email.com'
         self.data_api_client.get_framework.return_value = self.g8_framework
         self.data_api_client.get_supplier_framework_info.return_value = same_email_as_current_user
+        mocked_notify_client = mocked_notify_class.return_value
+        mocked_notify_client.templates = {'g-cloud-8_variation_1': 123456789}
         self.client.post(
             "/suppliers/frameworks/g-cloud-8/contract-variation/1",
             data={"accept_changes": "Yes"}
         )
 
-        mandrill_send_email.assert_called_once_with(
-            mock.ANY,  # self
-            to_email_addresses=['email@email.com'],
-            email_body=mock.ANY,
-            subject='G-Cloud 8: you have accepted the proposed contract variation',
-            from_email_address='enquiries@digitalmarketplace.service.gov.uk',
-            from_name='Digital Marketplace Admin',
-            tags=['g-cloud-8-variation-accepted'],
+        mocked_notify_client.send_email.assert_called_once_with(
+            to_email_address='email@email.com',
+            personalisation={'framework_name': 'g-cloud-8'},
+            template_name_or_id=123456789
         )
 
-    def test_success_message_is_displayed_on_success(self):
+    @mock.patch('app.main.views.frameworks.DMNotifyClient', autospec=True)
+    def test_success_message_is_displayed_on_success(self, mocked_notify_class):
+        mocked_notify_client = mocked_notify_class.return_value
+        mocked_notify_client.templates = {'g-cloud-8_variation_1': 123456789}
         self.data_api_client.get_framework.return_value = self.g8_framework
         self.data_api_client.get_supplier_framework_info.return_value = self.good_supplier_framework
         res = self.client.post(
```

Comment:
The test below this one (`test_api_is_not_called_and_no_email_sent_for_subsequent_posts`) still has a `DMMandrillClient` patch - it needs replacing with the Notify client so we can assert the email isn't sent.